src/osd/OSD.cc

```diff
@@ -5834,8 +5834,40 @@ bool OSD::scrub_random_backoff()
   return false;
 }
 
+bool OSD::scrub_time_permit()
```

Comment:
ceph_clock_now() actually already takes a cct now and can add in a time offset.  but we don't use it and not all callers pass the cct.  but a simpler test would be to just look at the current time and set a scrub range that either does or doesn't include now and verify that scrub_time_permit() returns the right value.